tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs

```diff
@@ -131,9 +131,45 @@ public static Result IsAspNetCoreMvcV1(this MockSpan span) => Result.FromSpan(sp
                 .Matches("component", "aspnet_core")
                 .Matches("span.kind", "server"));
 
-        public static Result IsAwsSqsV1(this MockSpan span) => Result.FromSpan(span)
+        public static Result IsAwsSqsInboundV1(this MockSpan span) => Result.FromSpan(span)
+                                                                            .Properties(s => s
+                                                                                            .Matches(Name, "aws.sqs.process")
+                                                                                            .Matches(Type, "http"))
+                                                                            .Tags(s => s
+                                                                                      .Matches("aws.agent", "dotnet-aws-sdk")
+                                                                                      .IsPresent("aws.operation")
+                                                                                      .IsOptional("aws.region")
+                                                                                      .IsPresent("aws.requestId")
+                                                                                      .Matches("aws.service", "SQS")
+                                                                                      .IsPresent("aws.queue.name")
+                                                                                      .IsOptional("aws.queue.url")
+                                                                                      .IsPresent("http.method")
+                                                                                      .IsPresent("http.status_code")
+                                                                                      .IsPresent("http.url")
+                                                                                      .Matches("component", "aws-sdk")
+                                                                                      .Matches("span.kind", "client"));
+
+        public static Result IsAwsSqsOutboundV1(this MockSpan span) => Result.FromSpan(span)
+            .Properties(s => s
+                .Matches(Name, "aws.sqs.send")
+                .Matches(Type, "http"))
+            .Tags(s => s
+                .Matches("aws.agent", "dotnet-aws-sdk")
+                .IsPresent("aws.operation")
+                .IsOptional("aws.region")
+                .IsPresent("aws.requestId")
+                .Matches("aws.service", "SQS")
+                .IsPresent("aws.queue.name")
+                .IsOptional("aws.queue.url")
+                .IsPresent("http.method")
+                .IsPresent("http.status_code")
+                .IsPresent("http.url")
+                .Matches("component", "aws-sdk")
+                .Matches("span.kind", "client"));
```

Comment:
`span.kind` should depend on whether it's inbound, outbound or request, right? 🤔 

Seeing the `v1.md` file (which has different span.kinds unlike the rules) makes me think you updated it manually, take a look at [this function](https://github.com/DataDog/dd-trace-dotnet/blob/4bafc78fa25382fefb03fa4e052905ca371d1ffa/tracer/build/_build/Build.Utilities.cs#L191) that you can call with `build.sh GenerateSpanDocumentation`

Reply:
I did updated it manually. I didn't know the existence of that option.
Thank you Pablo!

Reply:
And yes, `span.kind` should be defined accordingly. Nice catch!